src/client/components/date-range-input/date-range-input.tsx

```diff
@@ -36,14 +36,10 @@ export interface DateRangeInputState {
 }
 
 export class DateRangeInput extends React.Component<DateRangeInputProps, DateRangeInputState> {
-
-  constructor(props: DateRangeInputProps) {
-    super(props);
-    this.state = {
-      dateString: "",
-      timeString: ""
-    };
-  }
+  state = {
```

Comment:
Missing type of state. Typescript does not always infer type of state field. I wonder why.